rclcpp/test/rclcpp/test_node.cpp

```diff
@@ -796,6 +818,19 @@ TEST_F(TestNode, has_parameter) {
   }
 }
 
+TEST_F(TestNode, list_parameters) {
+  auto node = std::make_shared<rclcpp::Node>("test_list_parameter_node"_unq);
+  {
```

Comment:
Do we need this extra scope?  It seems like everything will go out of scope properly without it, but I may be missing something.

Reply:
Fixed. (result of copy-paste from unit test just above this one)